bigframes/ml/cluster.py

```diff
@@ -34,6 +34,7 @@
     "distance_type": "distanceType",
     "max_iter": "maxIterations",
     "tol": "minRelativeProgress",
+    "warm_start": "warmStart",
```

Comment:
We should add the unit test for all these newly-added params. Same for all other BQML models, but feel free to address in a follow-up PR.

Reply:
yes, will do a follow up.